### DISCUSSION TITLE
How to add extensions

### DISCUSSION THREAD

**Discussion by tintong** [PSAppDeployToolkit/PSAppDeployToolkit#1172] [Q&A] [answered]
Hello,

I have some extensions in my previous PSADT apps that I'm trying to make use of in the new PSADT version.
How can I add them please because in the PSAppDeployToolkit.Extension folder, I can only see psd1 or a psm1 file and I'm not sure which one I should be using please?

Many thanks

**Comment by mjr4077au** [comment, marked as answer]
Your code should go into the psm1 file, this is the scriptblock for the PSAppDeployToolkit.Extensions pre-provided module.
